pkg/console/operator/operator.go

```diff
@@ -175,10 +174,8 @@ func (c *consoleOperator) Sync(obj metav1.Object) error {
 
 	// all configs needed to do a sync
 	if err := c.handleSync(operatorConfig, consoleConfig, infrastructureConfig); err != nil {
-		c.SyncStatus(c.ConditionFailing(operatorConfig, "SyncLoopError", "Operator sync loop failed to completele."))
```

Comment:
It looks like we never set `Failing: True` after this PR?

Reply:
Or is that what "Update for Failed coming" means? :)

Reply:
Yeah the `/hold` means more is coming :)

Reply:
sorry i thought it was about the failed -> degraded changes. carry on :)